client/hotrod-client/src/test/java/org/infinispan/client/hotrod/ClientConnectionPoolingTest.java

```diff
@@ -178,29 +172,19 @@ public void testMaxActiveReached() throws Exception {
          workerThread2.putAsync("k4", "v4");
          log.info("Async calls for k3 and k4 is done.");
 
-         eventually(new Condition() {
-            @Override
-            public boolean isSatisfied() throws Exception {
-               return 1 == connectionPool.getNumActive(hrServ1Addr) &&
-               1 == connectionPool.getNumActive(hrServ2Addr) &&
-               0 == connectionPool.getNumIdle(hrServ1Addr) &&
-               0 == connectionPool.getNumIdle(hrServ2Addr);
-            }
-         });
+         eventually(() -> 1 == connectionPool.getNumActive(hrServ1Addr) &&
+         1 == connectionPool.getNumActive(hrServ2Addr) &&
+         0 == connectionPool.getNumIdle(hrServ1Addr) &&
+         0 == connectionPool.getNumIdle(hrServ2Addr));
 
 
          // another operation for each server, creating new connections
          workerThread3.putAsync("k5", "v5");
          workerThread4.putAsync("k6", "v6");
-         eventually(new Condition() {
-            @Override
-            public boolean isSatisfied() throws Exception {
-               return 2 == connectionPool.getNumActive(hrServ1Addr) &&
-                     2 == connectionPool.getNumActive(hrServ2Addr) &&
-                     0 == connectionPool.getNumIdle(hrServ1Addr) &&
-                     0 == connectionPool.getNumIdle(hrServ2Addr);
-            }
-         });
+         eventually(() -> 2 == connectionPool.getNumActive(hrServ1Addr) &&
+               2 == connectionPool.getNumActive(hrServ2Addr) &&
+               0 == connectionPool.getNumIdle(hrServ1Addr) &&
+               0 == connectionPool.getNumIdle(hrServ2Addr));
```

Comment:
Note for the future: you can use `eventuallyEquals(2, () -> connectionPool.getNumActive(hrServ1Addr))` for a nicer exception message.